samples/snippets/csharp/concepts/linq/query-expression-basics_7.cs

```diff
@@ -0,0 +1,27 @@
+            int highestScore =
+                (from score in scores
+                 select score)
+                .Max();
+
+            // or split the expression
+            IEnumerable<int> scoreQuery =
+                from score in scores
+                select score;
+
+            int highScore = scoreQuery.Max();
```

Comment:
Isn't that what the first example does?
int highestScore =
                (from score in scores
                 select score)
                .Max();